src/components/Button/Button.jsx

```diff
@@ -24,6 +24,7 @@ const {
  */
 const Button = createClass({
 	displayName: 'Button',
+	propName: 'Button',
```

Comment:
makes it so that when u call `findTypes(props, Button)` it will also look for the prop `Button` on the props object in addition to `props.children`.